server/service/microsoft_mdm.go

```diff
@@ -541,6 +541,7 @@ func GetEncodedBinarySecurityToken(typeID fleet.WindowsMDMEnrollmentType, payloa
 	pld.Type = typeID
 
 	if typeID == fleet.WindowsMDMProgrammaticEnrollmentType {
+		pld.Payload.OrbitNodeKey = payload
 		pld.Payload.HostUUID = payload
```

Comment:
This should be removed, the payload cannot be both the orbit node key and the host uuid (in fact the host uuid should probably be removed, but should be double-checked with @marcosd4h ).